src/frontend/shared-components/src/components/table/Table.stories.js

```diff
@@ -1,28 +1,29 @@
 import React from "react";
 
 import Table from "./Table";
-import { getTableTestData } from "../../modules/tableTestData";
+import { getTableElementsTestData } from "../../modules/tableTestData";
 
 export default {
   title: "Table",
   component: Table
 };
 
-const tableData = getTableTestData();
+const tableData = getTableElementsTestData();
+const header = "BCeID Info";
 
-export const WithHeader = () => <Table table={tableData} />;
+export const WithHeader = () => <Table heading={header} elements={tableData} />;
```

Comment:
can we add `styling` to at least one of these? so that we have a snapshot where className is not ""
same for `id`